setup.py

```diff
@@ -9,6 +8,15 @@
 from shutil import rmtree
 
 from setuptools import Command, find_packages, setup
+from setuptools import __version__ as setuptools_version
```

Comment:
unless you have upstreamed your changes to pypi-mobans, your changes here will be short lived.

I could see you have your own local version of pypi-mobans.

Reply:
Of course I upstreamed my changes to pypi-mobans, and I submitted a PR already.

https://github.com/moremoban/moban/issues/286 for a feature to make this an integral part of moban.

Reply:
yep, I have merged your PR. I failed to notice your PR at my time of commenting.